PowerShell/JumpCloud Commands Gallery/Windows Commands/Windows - Forget RADIUS WiFi Network.md

```diff
@@ -0,0 +1,141 @@
+#### Name
+
+Windows - Forget RADIUS WiFi Network | v1.0 JCCG
+
+
+#### commandType
+
+windows
+
+#### Command
+
+```
+# Enter the SSID of the Radius network
+$RadiusSSID = ''
+
+# Change Overwrite to $true if you would like to re-create existing Scheduled Tasks
+# This should only be used if redistributing this command to devices
+$Overwrite = $false
+
+
+#### DO NOT EDIT BELOW THIS LINE ####
+$removeWifips1 = @"
+function Get-WifiProfile {
+    [cmdletbinding()]
+    param
+    (
+         [System.Array]`$Name = `$NULL
+    )
+    Begin {
+        `$list = ((netsh.exe wlan show profiles) -match '\s{2,}:\s') -replace '.*:\s' , ''
+        `$ProfileList = `$List | Foreach-object { [pscustomobject]@{Name = $_ } }
+    }
+    Process {
+         Foreach (`$WLANProfile in `$Name) {
+            `$ProfileList | Where-Object { `$_.Name -match `$WLANProfile }
+         }
+
+    }
+    End {
+         If (`$Name -eq `$NULL) {
+            `$Profilelist
+         }
+    }
+}
+function Remove-WifiProfile {
+    [cmdletbinding()]
+    param
+    (
+         [System.Array]`$Name = `$NULL
+    )
+    begin {}
+    process {
+         Foreach (`$item in `$Name) {
+            `$Result = (netsh.exe wlan delete profile `$item)
+              If (`$Result -match 'deleted') {
+                   "WifiProfile : `$Item Deleted"
+              } else {
+                   "WifiProfile : `$Item NotFound"
+              }
+         }
+    }
+}
+
+Remove-WifiProfile $RadiusSSID
```

Comment:
```suggestion
Remove-WifiProfile "$($RadiusSSID)"
```

for networks with spaces in the name this would be necessary

Reply:
Included this, good catch